### DISCUSSION TITLE
Bundle.css not being created when build is run?

### DISCUSSION THREAD

**Discussion by sinnyour** [uicrooks/shopify-theme-lab#51] [General]
After adding react to the starter project no css is compiled when running dev or at build no bundle.css file is created

**Comment by sergejcodes** [comment]
Hi, there 👋

Seems to be working for me with the react setup. 
If I run the `npm run build` task I get the `bundle.css` file emitted to the assets directory.

Could it be you don't have the CSS import inside your `main.js` file?

```js
import './css/main.css'
```
